packages/client/screens/SigningInScreen.tsx

```diff
@@ -223,6 +240,17 @@ const SigningInScreen: React.FC<SigningInScreenProps> = ({ navigation }) => {
                                     }}
                                 />
 
+                                <TouchableOpacity
+                                    style={sx({
+                                        marginTop: 'm',
+                                    })}
+                                    onPress={handleRequestPasswordReset}
+                                >
+                                    <Text sx={{ color: 'white' }}>
+                                        Do you lost your password?
```

Comment:
nitpick: Unless I'm mistaken should this be `Did you lost your password ?` or maybe we could be saying something like `Reset password`

Reply:
I made a typo you're right, but I think it should be _Did you lose your password_? Or _Have you lost your password_?

Reply:
Indeed :laughing: